### ISSUE TITLE
Question about the adv_loss

### ISSUE THREAD

**Issue by Damoners** [hongzimao/decima-sim#28]
Hello, I have a question about the adv_loss. 
In your paper, the value of the advantage part in the network parameter update formula is normal.
![image](https://user-images.githubusercontent.com/55888954/101624245-a122ac00-3a54-11eb-92dc-66d90e90b06b.png)

But why does **self.adv** take the opposite value when calculating **self.adv_loss** in the code? （in actor_agent.py）
In addition, what does the "# actor loss due to advantage (negated)" in the code comment "**negated**" mean?
```python
# actor loss due to advantage (negated)
self.adv_loss = tf.reduce_sum(tf.multiply(
            tf.log(self.selected_node_prob * self.selected_job_prob + \
                   self.eps), -self.adv))
```

Thank you!

**Comment by hongzimao** [comment]
When computing the gradient and updating the parameter, normally that leads to *minimize* the objective function. Notice that here we want to maximize the reward (hence the plus in the gradient in the equation in your screenshot). As a result, we need to reverse the adv value. Hope this helps!
